fastify.d.ts

```diff
@@ -111,6 +111,7 @@ declare namespace fastify {
     requestIdLogLabel?: string;
     jsonShorthand?: boolean;
     genReqId?: (req: RawRequestDefaultExpression<RawServer>) => string,
+    createRequestLogMessage?: (req: RawRequestDefaultExpression<RawServer>) => string,
```

Comment:
I believe it receives the `FastifyRequest` type instead of `http.IncomingMessage`, which is from where the `RawRequestDefaultExpression<RawServer>` is based on

Reply:
Ok no problem, I will change it to FastifyRequest and have it return a string.

Reply:
Is it FastifyRequest or the RawRequest?

Reply:
Here's how I set the prototype:

createRequestLogMessage?: (ChildLoggerOptions: ChildLoggerOptions, FastifyRequest) => string

Please let me know if you guys have any alternative suggestions.

Reply:
Yeah, it should be `FastifyRequest`. We already constructed one at the moment we are printing the `incoming request` log